pywinauto/mouse.py

```diff
@@ -238,8 +239,24 @@ def right_click(coords=(0, 0)):
     _perform_click_input(button='right', coords=coords)
 
 
-def move(coords=(0, 0)):
+def move(coords=(0, 0), duration=0):
     """Move the mouse"""
+    minimum_duration = 0.05
+    if duration >= minimum_duration:
+        coord_start = win32api.GetCursorPos()
+        delta_x = coords[0] - coord_start[0]
+        delta_y = coords[1] - coord_start[1]
+        max_delta = max(abs(delta_x), abs(delta_y))
+        num_steps = max_delta
+        sleep_amount = duration / max(num_steps, 1)
+        if sleep_amount < minimum_duration:
+            num_steps = int(num_steps * sleep_amount / minimum_duration)
+            sleep_amount = minimum_duration
+        delta_x /= max(num_steps, 1)
+        delta_y /= max(num_steps, 1)
+        for step in range(0, num_steps):
```

Comment:
`range(num_steps)` is enough.

Reply:
fixed, 33510e84146ff80bc29bb0f5e8adddb85c73fdbb